TestPlaybooks/script-TestPyCommonServer.yml

```diff
@@ -337,6 +337,10 @@ script: |-
       assert camelize([{'chookity_bop': 'asdasd'}, {'ab_c': 'd e', 'fgh_ijk': 'lm', 'nop': 'qr_st'}], '_').__str__ == "[{u'ChookityBop': 'asdasd'}, {u'AbC': 'd e', u'Nop': 'qr_st', u'FghIjk': 'lm'}]"
       assert camelize({'ab_c': 'd e', 'fgh_ijk': 'lm', 'nop': 'qr_st'}, '_').__str__ == "{u'AbC': 'd e', u'Nop': 'qr_st', u'FghIjk': 'lm'}"
 
+  def test_date_to_timestamp():
```

Comment:
I think that it should be mentioned in the TESTS dict in order to be tested, Also saw that CAMELIZE is not there so please add both of them

Reply:
Done